EapImport/src/main/java/de/hpi/unicorn/importer/json/JsonExporter.java

```diff
@@ -88,4 +89,42 @@ public static File generateExportFileWithDependencies(final EapEventType eventTy
         }
         return file;
     }
+
+    public static File generateExportFileWithValues(EapEventType eventType, Map<TypeTreeNode, String> inputValues, int eventCount, int scaleFactor, String timestamp) {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 167 characters long line (which is greater than 150 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS00103)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Missing a Javadoc comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AJavadocMethod_BP2016)